packages/web/src/scss/components/Item/_Item.scss

```diff
@@ -0,0 +1,85 @@
+@use 'sass:map';
+@use 'theme';
+@use '../../settings/cursors';
+@use '../../tools/typography';
+
+.Item {
+    display: grid;
+    grid-template-columns: auto 1fr auto;
+    padding: theme.$padding;
+    border-radius: theme.$border-radius;
+    background-color: theme.$background-default;
+    cursor: pointer;
+
+    @media (hover: hover) {
+        &:hover {
+            text-decoration: none;
+            background-color: theme.$background-hover;
+        }
+
+        &:active {
+            text-decoration: none;
+            background-color: theme.$background-active;
+        }
+    }
+}
+
+.Item__label {
+    @include typography.generate(theme.$label-typography);
+
+    display: inline-block;
+    grid-column: 2;
+    color: theme.$label-color-default;
```

Comment:
I'm wondering if we could reuse the existing `form-fields.item-*()` mixins. If I get it right, `Item` styling is now duplicated: it's here, and in `settings/_form-fields.scss` and `tools/_form-fields.scss`.

As long as we want to get the same visual outcome (do we?), I believe we should reuse these settings and mixins, even if it means we would eventually delete them and move here.

What do you think?